fimsauthz-api/swagger.yaml

```diff
@@ -16,20 +16,32 @@ securityDefinitions:
   application:
     type: oauth2
     # Should be updated to the Token URL of your OAuth provider.
-    tokenUrl: https://thisserver.com/oauth/token
+    tokenUrl: https://utm.defined.host/oauth/token
     flow: application
     scopes:
-      'utm.nasa.gov_read.constraint': Subject can read UTM constraint data
-      'utm.nasa.gov_write.constraint': Subject can read, create, and update UTM constraint data. This means that the Subject can define areas that restrict other operations.
-      'utm.nasa.gov_read.min_data': Subject can read data which has been minimized for example, by redacting some properties
-      'utm.nasa.gov_write.operation': Subject can read, create, and update operational data such as USS Instance data
+      'utm.nasa.gov_read.fimsadmin': Subject can read data internal to FIMS.
+      'utm.nasa.gov_write.fimsadmin': Subject can read, create, and update data internal to FIMS.
       'utm.nasa.gov_read.operation': Subject can read operational data such as USS Instance data
-      'utm.nasa.gov_write.message': Subject can read, create, and update message data such as UTM Message and Negotiations
+      'utm.nasa.gov_write.operation': Subject can read, create, and update operational data such as USS Instance data
       'utm.nasa.gov_read.message': Subject can read message data such as UTM Message and Negotiations
+      'utm.nasa.gov_write.message': Subject can read, create, and update message data such as UTM Message and Negotiations
+      'utm.nasa.gov_read.publicsafety': Subject can read operations that are designated public safety operations.
       'utm.nasa.gov_write.publicsafety': Subject can read, create, and update operations that are designated public safety operations.
+      'utm.nasa.gov_read.constraint': Subject can read UTM constraint data
+      'utm.nasa.gov_write.constraint': Subject can read, create, and update UTM constraint data. This means that the Subject can define areas that restrict other operations.
+      'utm.nasa.gov_read_min.uvin': Subject can read redacted vehicle registration data.
+      'utm.nasa.gov_read.uvin':  Subject can read vehicle registration data.
+      'utm.nasa.gov_write.uvin':  Subject can read and write vehicle registration data.
+      'utm.nasa.gov_read.stategic_cm':  Subject can read strategic conflict management data.
```

Comment:
I think this is why you were concerned about naming conventions with "strategic deconfliction" and "strategic conflict management"... correct?
Would like to avoid the snake_case in the suffix.
perhaps `utm.nasa.gov_read.conflictmanagement`?  this may allow for messaging that isn't limited to the strategic time scale and avoids the distinction mentioned above.
I am not sold on this... happy to discuss further.

Reply:
>>"strategic deconfliction" and "strategic conflict management"... correct
correct

>> utm.nasa.gov_read.conflictmanagement 
great idea!  I was trying to get a good name.